projects/igniteui-angular/src/lib/grid/column-moving.spec.ts

```diff
@@ -0,0 +1,1391 @@
+import { DebugElement } from '@angular/core';
+import { async, fakeAsync, TestBed, tick } from '@angular/core/testing';
+import { FormsModule } from '@angular/forms';
+import { By } from '@angular/platform-browser';
+import { BrowserAnimationsModule, NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { IgxGridModule } from './index';
+import { IgxColumnGroupComponent } from './column.component';
+import { IgxInputDirective } from '../directives/input/input.directive';
+import { IgxToggleDirective } from '../directives/toggle/toggle.directive';
+import { SortingDirection } from '../data-operations/sorting-expression.interface';
+import {
+    MovableColumnsComponent,
+    MovableTemplatedColumnsComponent,
+    MovableColumnsLargeComponent,
+    MultiColumnHeadersComponent
+ } from '../test-utils/grid-samples.spec';
+import { UIInteractions, wait } from '../test-utils/ui-interactions.spec';
+
+const CELL_CSS_CLASS = '.igx-grid__td';
+const COLUMN_HEADER_CLASS = '.igx-grid__th';
+const COLUMN_GROUP_HEADER_CLASS = '.igx-grid__th--fw';
+
+describe('IgxGrid - Column Moving', () => {
+    beforeEach(async(() => {
+        TestBed.configureTestingModule({
+            declarations: [
+                MovableColumnsComponent,
+                MovableTemplatedColumnsComponent,
+                MovableColumnsLargeComponent,
+                MultiColumnHeadersComponent
+            ],
+            imports: [
+                FormsModule,
+                BrowserAnimationsModule,
```

Comment:
With NoopAnimationsModule included, the BrowserAnimationsModule should not be necessary.